src/sage/modular/multiple_zeta_F_algebra.py

```diff
@@ -11,6 +11,9 @@
 Here we provide a basic direct implementation, endowed with the
 motivic coproduct.
 
+The similar algebra on generators `f_1`, `f_3`, `f_5`, ... and `f_2`
+is now also available.
```

Comment:
```suggestion
The similar algebra with generators a subset of
`f_1`, `f_3`, `f_5`, `\ldots` and `f_2` is now also available.
```
Perhaps this is closer to what you mean? The current addition makes it seem like it is just the same algebra as it is generated by the same set (unless I am missing something).

Reply:
no, but I wrote something vague, for sure. There is a base ring which remains a polynomial in f_2, over which one has a shuffle algebra in one more generator f_1, in addition to the generators f_3, f_5, etc. Should I reformulate ?

Reply:
However, the code is saying something slightly different: you want to consider the algebras that have $f_2$ and $f_i, f_{i+2}, \ldots$ for some odd integer $i$. Of course, this includes $i = 1$ as per the ticket description, but I guess there is a slight mismatch between your intent and implementation? Perhaps it would be better to rename `start` (at least for the algebra input) to `extended` or some other boolean input to match this documentation?